tests/pytests/unit/utils/test_network.py

```diff
@@ -1,8 +1,1535 @@
+        assert ret == expected_mac
+
+
+def test_mac_str_to_bytes():
+    pytest.raises(ValueError, network.mac_str_to_bytes, "31337")
+    pytest.raises(ValueError, network.mac_str_to_bytes, "0001020304056")
+    pytest.raises(ValueError, network.mac_str_to_bytes, "00:01:02:03:04:056")
+    pytest.raises(ValueError, network.mac_str_to_bytes, "a0:b0:c0:d0:e0:fg")
+    assert b"\x10\x08\x06\x04\x02\x00" == network.mac_str_to_bytes("100806040200")
+    assert b"\xf8\xe7\xd6\xc5\xb4\xa3" == network.mac_str_to_bytes("f8e7d6c5b4a3")
+
+
+@pytest.mark.slow_test
+def test_generate_minion_id_with_long_hostname():
+    """
+    Validate the fix for:
+
+    https://github.com/saltstack/salt/issues/51160
+    """
+    long_name = "localhost-abcdefghijklmnopqrstuvwxyz-abcdefghijklmnopqrstuvwxyz"
+    with patch("socket.gethostname", MagicMock(return_value=long_name)):
+        # An exception is raised if unicode is passed to socket.getfqdn
+        minion_id = network.generate_minion_id()
+    assert minion_id != "", minion_id
+
+
+def test_filter_by_networks_with_no_filter():
+    ips = ["10.0.123.200", "10.10.10.10"]
+    with pytest.raises(TypeError):
+        network.filter_by_networks(ips)  # pylint: disable=no-value-for-parameter
+
+
+def test_filter_by_networks_empty_filter():
+    ips = ["10.0.123.200", "10.10.10.10"]
+    assert network.filter_by_networks(ips, []) == []
+
+
+def test_filter_by_networks_ips_list():
+    ips = [
+        "10.0.123.200",
+        "10.10.10.10",
+        "193.124.233.5",
+        "fe80::d210:cf3f:64e7:5423",
+    ]
+    networks = ["10.0.0.0/8", "fe80::/64"]
+    assert network.filter_by_networks(ips, networks) == [
+        "10.0.123.200",
+        "10.10.10.10",
+        "fe80::d210:cf3f:64e7:5423",
+    ]
+
+
+def test_filter_by_networks_interfaces_dict():
+    interfaces = {
+        "wlan0": ["192.168.1.100", "217.5.140.67", "2001:db8::ff00:42:8329"],
+        "eth0": [
+            "2001:0DB8:0:CD30:123:4567:89AB:CDEF",
+            "192.168.1.101",
+            "10.0.123.201",
+        ],
+    }
+    assert network.filter_by_networks(
+        interfaces, ["192.168.1.0/24", "2001:db8::/48"]
+    ) == {
+        "wlan0": ["192.168.1.100", "2001:db8::ff00:42:8329"],
+        "eth0": ["2001:0DB8:0:CD30:123:4567:89AB:CDEF", "192.168.1.101"],
+    }
+
+
+def test_filter_by_networks_catch_all():
+    ips = [
+        "10.0.123.200",
+        "10.10.10.10",
+        "193.124.233.5",
+        "fe80::d210:cf3f:64e7:5423",
+    ]
+    assert ips == network.filter_by_networks(ips, ["0.0.0.0/0", "::/0"])
+
+
+def test_ip_networks():
+    # We don't need to test with each platform's ifconfig/iproute2 output,
+    # since this test isn't testing getting the interfaces. We already have
+    # tests for that.
+    interface_data = network._interfaces_ifconfig(LINUX)
+
+    # Without loopback
+    ret = network.ip_networks(interface_data=interface_data)
+    assert ret == ["10.10.8.0/22"], ret
```

Comment:
```suggestion
    assert ret == ["10.10.8.0/22"]
```
The same for the remaining assertions

Reply:
done